### DISCUSSION TITLE
Bigquery output plugin support

### DISCUSSION THREAD

**Discussion by shennongjushi** [fluent/fluent-bit#5523] [Q&A]
Hi all,

I am going through the bigquery output plugin document and have some questions:
(1) The document mentioned the "Application Default Credentials" is not supported for now, may I know when would this feature be supported?
I have tried the Fluentd bigquery output plugin before, I use work identity in GKE and set with Application Default Credentials, it works. I am wondering for Fluent bit, without such support, what other ways I could use? (We have policies to avoid downloaded/exported SA Keys)

(2) For the configuration, for example:
[INPUT]
    Name  dummy
    Tag   dummy

[OUTPUT]
    Name       bigquery
    Match      *
    dataset_id my_dataset
    table_id   dummy_table

Could the "dataset_id", "table_id", "project_id" be placeholder? For example, if I could use information in the tag to assign different value to it so it could be output to different Bigquery tables?

Thanks!

**Comment by agup006** [comment]
@shennongjushi thanks for adding the discussion, we don't have any current plans for expanding BigQuery featureset. Would you be open to contributing this feature?
